# JCP300 - RAC
uint32 turbine_rpm
int32 egt_temp
float32 pump_voltage
uint8 turbine_state
float32 throttle_position
float32 engine_current